### ISSUE TITLE
I have import error at 'seleniumhelper'

### ISSUE THREAD

**Issue by HanChangHun** [thelightone/TradingViewScrape#1]
Hello, I am interested in your code.
But there is library that cannot be imported, so I'm asking you a question.
```python
from seleniumhelper import *
```
Where can I get the above library?

Thank you!

**Comment by kokosandim** [comment]
The same problem

**Comment by kokosandim** [comment]
> Hello, I am interested in your code.
> But there is library that cannot be imported, so I'm asking you a question.
> 
> ```python
> from seleniumhelper import *
> ```
> 
> Where can I get the above library?
> 
> Thank you!

Hey! Did you find anything?

**Comment by kokosandim** [comment]
> Hello, I am interested in your code.
> But there is library that cannot be imported, so I'm asking you a question.
> 
> ```python
> from seleniumhelper import *
> ```
> 
> Where can I get the above library?
> 
> Thank you!

I think you should follow the command "pip install selenium-helper"
After that change 19 string to "selenium_helper", already I see different errors

**Comment by HanChangHun** [comment]
No, I haven't found anything yet either. Maybe it's a self-made package.
